mlflow/utils/autologging_utils/__init__.py

```diff
@@ -543,4 +543,8 @@ def should_log(self):
                 self._parent.allow_children and self._parent.clazz != self.clazz
             )
 
+        @staticmethod
+        def in_top_level():
```

Comment:
Perhaps `is_active()` may be a more descriptive name, as it indicates whether or not autologged training is actively happening. Can we also add docs for this?

```suggestion
        def is_active():
```

Reply:
and a small test for this in `test_training_session.py`?